README.md

```diff
@@ -24,9 +24,43 @@ Please be advised to change any automated implementation you might have that is
 ## Installation
 `tfswitch` is available as a binary and on various package managers (eg. Homebrew). 
 
-See [our installation documentation](https://tfswitch.warrensbox.com/Installation) for more details.
 
-> [!IMPORTANT]
+## Windows
+Download and extract the Windows version of `tfswitch` that is compatible with your system.  
+We are building binaries for 386, amd64, arm6 and arm7 CPU structure.  
+See the [release page](https://github.com/warrensbox/terraform-switcher/releases/latest) for your download.
+
+## Homebrew
+Installation for macOS is the easiest with Homebrew. <a href="https://brew.sh/" target="_blank">If you do not have homebrew installed, click here</a>.
```

Comment:
That's not essential in this case, though Homebrew in not just for macOS users. I have `tfswitch` (and many other tools/software) managed by Homebrew on Ubuntu.